challenge/src/test/java/com/ifood/challenge/ChallengeApplicationTest.java

```diff
@@ -0,0 +1,26 @@
+package com.ifood.challenge;
+
+import com.ifood.challenge.location.Location;
+import com.ifood.challenge.location.LocationService;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ChallengeApplicationTest extends LocationService {
```

Comment:
A test is always independent from any other system resources...

So, you cannot extend `LocationService` here.
What you're saying is: my test is a location service. Does it makes sense?

The exercise required an integration test, which is a very specific kind of test.

So I suggest you understand some boundaries like:

- Unit test
- Integration test

And some resources you're gonna need to properly execute integration tests on Spring:

- `@SpringBootTest` annotation
- `@Autowire` to provide dependency injection into your test. Knowing what dependency injection is, is a good start also

Reply:
You can check `@MockMvc` also.